config/regex-url-follow.txt

```diff
@@ -0,0 +1,4 @@
+# 1. extract url will be named capture group 'url',
+# 2. url to be prepended to what is extracted from href, if none 2 spaces
```

Comment:
should we use something more explicit here instead of two spaces?

Reply:
thought of it but doesn't really seem necessary for now